lib/css/components/_stacks-activity-indicator.less

```diff
@@ -13,9 +13,17 @@
 
 .s-activity-indicator {
     display: inline-block;
-    width: @su12;
+    text-align: center;
```

Comment:
I don't think this `text-align` is needed. We could save ~20 bytes if you never see this being multi-line.

```suggestion
```